tests/Microsoft.DotNet.Docker.Tests/DockerHelper.cs

```diff
@@ -79,16 +79,23 @@ public string BuildDistrolessHelper(DotNetImageRepo imageRepo, ProductImageData
             // version of the runtime-deps image get the correct image.
             ProductImageData runtimeDepsImageData = new()
             {
-                Version = imageData.VersionFamily,
+                // Special case for .NET 8.0 Aspire Dashboard images - the Dashboard is in preview even though
+                // .NET 8.0 is not. The distroless helper image should not be built with the preview version.
+                Version = imageData.Version == ImageVersion.V8_0_Preview ? ImageVersion.V8_0 : imageData.Version,
```

Comment:
Shouldn't it be like this to match the original logic?
```suggestion
                Version = imageData.Version == ImageVersion.V8_0_Preview ? ImageVersion.V8_0 : imageData.VersionFamily,
```